.github/workflows/appengine-deploy.yml

```diff
@@ -0,0 +1,47 @@
+name: App Engine Deploy Integration
+
+on: 
+  push:
+    branches:
+      - main 
+  pull_request:
+
+env:
+  APPENGINE_VERSION: invalid
+
+jobs:
+  appengine-deploy:
+    name: App Engine Deploy Integration
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+
+      - id: vars
+        name: Declare sha_short as a variable
+        shell: bash
+        run: echo "::set-output name=sha_short::$(git rev-parse --short=7 ${{ github.sha }})"
```

Comment:
We use both `set-env` and `set-output`. Is there a reason why need to use both? Any reason to only use `set-env` that manipulates output parameters?